package com.voterapp.exception;

public class NotEligibleException  extends Exception{
	
	private static final long serialVersionIUID=1L;
	
	public NotEligibleException() {
		super();
	}
	
	public NotEligibleException(String message) {
		super(message);
	}

}
